ImportExtension/AddImportOperation.swift

```diff
@@ -120,9 +120,12 @@ class AddImportOperation {
         self.buffer.lines.insert(importString, at: line)
         
         //add a new selection. Bug fix for #7
-        let selectionPosition = XCSourceTextRange.init(start: XCSourceTextPosition.init(line: 0, column: 0), end: XCSourceTextPosition.init(line: 0, column: 0))
```

Comment:
Sorry, I ment to say Xcode 8.3.x; after testing it on Xcode 8.3.3, I was having no issues.